### PR TITLE
doc: Add missing supported rpcs to doc/descriptors.md

### PR DESCRIPTION
**Pull request by andrewtoth**
Improve descriptor docs by adding missing rpcs.

### CODE REVIEW

**Comment by laanwj** [comment]
Thanks for adding documentation. The list looks good to me.
ACK b7982c92df44d0b526c5db57019a455edd933deb

**Comment by Sjors** [comment]
ACK  b7982c9, though there's also `generatetodescriptor` (regtest only).

**Comment by andrewtoth** [comment]
@Sjors I wasn't sure if I should add that since it's hidden.

**Comment by Sjors** [comment]
@andrewtoth it's not really hidden; if you start `bitcoind -regtest` it's listed with `bitcoin-cli help`. The audience for this particular document is probably developers.

File: doc/descriptors.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ Supporting RPCs are:
 - `deriveaddresses` takes as input a descriptor and computes the corresponding
   addresses.
 - `listunspent` outputs a specialized descriptor for the reported unspent outputs.
+- `getaddressinfo` - returns a descriptor for solvable addresses. (Since v0.18)
+- `importmulti` - takes as input an array of descriptors and imports them into
+  the wallet. (Since v0.18)
+- `utxoupdatepsbt` - takes as input an array of descriptors and adds information
+  to the psbt. (Since v0.19)
```
Comment (jonatack) [inline doc/descriptors.md]: nit x 3: s/. (Since v.0.19)/ (since v0.19)./

**Review by jonatack — COMMENTED** [review]
Concept ACK. I agree with @Sjors wrt https://github.com/bitcoin/bitcoin/pull/18025#issuecomment-580304568. Should the eight rpcs be listed in alphabetical order, or some other type of ordering?

**Comment by andrewtoth** [comment]
@Sjors Updated to add `generatetodescriptor`.

@jonatack I added them in order of earliest version to be supported, and then the order they appear in `help`.

**Comment by Sjors** [comment]
Thanks, ACK 5024291

**Comment by andrewtoth** [comment]
squashed

**Comment by emilengler** [comment]
ACK 7347fe57eef4b4da0cf9df1657b4688cb3aed54a

nit: Maybe it would be better to use `v0.19.0.1` to avoid confusion

**Comment by andrewtoth** [comment]
>nit: Maybe it would be better to use v0.19.0.1 to avoid confusion

@emilengler as Sjors pointed out this document is mainly for developers, so I doubt this will cause any confusion. It's also technically true, even though v0.19.0 didn't get an official release.

**Comment by Sjors** [comment]
ACK 7347fe5

File: doc/descriptors.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ Supporting RPCs are:
 - `deriveaddresses` takes as input a descriptor and computes the corresponding
   addresses.
 - `listunspent` outputs a specialized descriptor for the reported unspent outputs.
+- `getaddressinfo` - outputs a descriptor for solvable addresses (since v0.18).
+- `importmulti` - takes as input descriptors to import into the wallet
+  (since v0.18).
+- `generatetodescriptor` - takes as input a descriptor and generates coins to it
+  (`regtest` only, since v0.19).
+- `utxoupdatepsbt` - takes as input descriptors to add information to the psbt
```
Comment (jonatack) [inline doc/descriptors.md]: Apologies for not seeing it before, but the new entries add a hyphen (" - ") between the rpc name and the description which the existing entries do not have.

**Review by jonatack — COMMENTED** [review]
ACK 7347fe5 modulo comment (sorry, feel free to ignore). I think the major+minor release versions are sufficient and encompass all patch versions.

File: doc/descriptors.md (hunk 2 above)
Comment (andrewtoth) [reply doc/descriptors.md]: Huh wonder why I thought there were hyphens... anyways fixed.

**Review by fanquake — APPROVED** [review]
ACK c7ec9a18888e040a2e1de2cc740a6ef0372d336d - I think this has been bikeshed enough.

**Comment by jonatack** [comment]
ACK c7ec9a18888e040a2e1de2cc740a6ef0372d336d

**Comment by vova557** [comment]
Как можно получать деньги за свое по подскажите пожалуйста и помогите если можно я не давно программирую

**Comment by emilengler** [comment]
@vova557 Welcome to Bitcoin Core but please speak English so we can all understand you ;)

**Comment by vova557** [comment]
ЗДРАВСТВУЙТЕ Я ЗАНИМАЮСЬ РАЗРАБОТКОЙ ЛАЙТКОИНА И БИТКОИНА НО НЕ ПОЛУЧАЮ ЗА ЭТО ДЕНЬГИ КАК ПРОИСХОДИТ ОПЛАТА РАЗРАБОТЧИКАМ СПАСИБО ЗАРАНЕЕ 07.03.2020, 17:07, "Emil Engler" <notifications@github.com>:@vova557 Welcome to Bitcoin Core but please speak English so we can all understand you ;)—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by vova557** [comment]
HELLO I AM DEVELOPING LITECOIN AND BITCOIN BUT I DO NOT GET PAID FOR IT AS PAYMENT IS MADE TO DEVELOPERS THANK YOU IN ADVANCE07.03.2020, 17:32, "владимир архипов" <skylmaxarhipov@yandex.ru>:ЗДРАВСТВУЙТЕ Я ЗАНИМАЮСЬ РАЗРАБОТКОЙ ЛАЙТКОИНА И БИТКОИНА НО НЕ ПОЛУЧАЮ ЗА ЭТО ДЕНЬГИ КАК ПРОИСХОДИТ ОПЛАТА РАЗРАБОТЧИКАМ СПАСИБО ЗАРАНЕЕ 07.03.2020, 17:07, "Emil Engler" <notifications@github.com>:@vova557 Welcome to Bitcoin Core but please speak English so we can all understand you ;)—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request merged**